bridge_ui/src/utils/consts.ts

```diff
@@ -715,6 +715,7 @@ export const ALEPHIUM_MINIMAL_CONSISTENCY_LEVEL =
     : CLUSTER === "testnet"
     ? TESTNET_ALPH_MINIMAL_CONSISTENCY_LEVEL
     : DEVNET_ALPH_MINIMAL_CONSISTENCY_LEVEL;
+export const ALEPHIUM_ATTEST_TOKEN_CONSISTENCY_LEVEL = 10
```

Comment:
We don't need to wait for 105 confirmations for the attest token on the mainnet.